ansible/roles_ocp_workloads/ocp4_workload_external_odf/defaults/main.yml

```diff
@@ -1,5 +1,17 @@
 ---
 become_override: false
-odf_channel: "stable-4.13"
+odf_channel: stable-4.13
```

Comment:
I usually set default to current - as in stable-4.16. But that's just a preference on my part.